can-stache-bindings.js

```diff
@@ -142,27 +141,32 @@ var behaviors = {
 			});
 
 			if(dataBinding) {
-				bindingsState = checkBindingState(bindingsState, dataBinding);
+				var bindingInfo = dataBinding.bindingInfo;
+				bindingsState = checkBindingState(bindingsState, bindingInfo);
 				hasDataBinding = true;
 
-
 				// For bindings that change the viewModel,
-				if(dataBinding.onCompleteBinding) {
-					// save the initial value on the viewModel.
-					if(dataBinding.bindingInfo.parentToChild && dataBinding.value !== undefined) {
+				// save the initial value on the viewModel.
+				if(bindingInfo.parentToChild) {
+					var parentValue = bindingInfo.stickyParentToChild ? makeCompute(dataBinding.parent) : dataBinding.canBinding.parentValue;
```

Comment:
We discussed moving this to can-bind, but I decided not to (yet) because `makeCompute` comes from can-view-scope: https://github.com/canjs/can-view-scope/blob/master/make-compute-like.js

It seemed wrong to me to add can-view-scope as a dependency of can-bind.

This is the test that fails without this special handling: https://github.com/canjs/can-stache-bindings/blob/129e29b70fd108c1418957d5f68a043a025def1d/test/colon/view-model-test.js#L175

Reply:
This code also exists in `can-simple-observable`: https://github.com/canjs/can-simple-observable/blob/master/make-compute/make-compute.js.